puni/version.py

```diff
@@ -14,4 +14,4 @@
 with puni. If not, see http://www.gnu.org/licenses/.
 """
 
-__version__ = '1.1.2'
+__version__ = '1.1.4'
```

Comment:
```suggestion
__version__ = '1.1.3'
```

The version should only increment by 1